src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -2764,8 +2764,10 @@ public void testCreatePoWithDifferentVendorStatus() throws Exception {
     assertThat(activeVendorActiveAccessProviderErrors.getErrors(), empty());
 
     // Internal mod-vendor error
-    Errors internalServerError = verifyPostResponseErrors(1, MOD_VENDOR_INTERNAL_ERROR_ID, ACTIVE_ACCESS_PROVIDER_A, ACTIVE_ACCESS_PROVIDER_B);
-    assertEquals(GENERIC_ERROR_CODE.getCode(), internalServerError.getErrors().get(0).getCode());
+    Errors errors = verifyPostResponse(COMPOSITE_ORDERS_PATH, getPoWithVendorId(MOD_VENDOR_INTERNAL_ERROR_ID, ACTIVE_ACCESS_PROVIDER_A, ACTIVE_ACCESS_PROVIDER_B),
+      EXIST_CONFIG_X_OKAPI_TENANT_LIMIT_10, APPLICATION_JSON, 500).as(Errors.class);
+    assertEquals(errors.getTotalRecords(), new Integer(1));
```

Comment:
I think the `assertEquals(...)` args are in order (expected, actual), which is annoyingly the opposite of `assertThat(...)`...

Given this, and that `new Integer(1)` seems a little clumsy to me, I would suggest using the following instead:

`assertThat(errors.getTotalRecords(), equalTo(1));`